tests/Common/TokensTest.php

```diff
@@ -98,6 +98,7 @@ public function testTokenProperties()
     {
         $currentToken = $this->_client->verifyToken();
 
+        $this->assertArrayNotHasKey('token', $currentToken, 'Master token should not return unencrypted token');
```

Comment:
```suggestion
        $this->assertArrayNotHasKey('token', $currentToken);
```

Reply:
V dávné minulosti jsme si říkali že psát message do assertů nebudeme.

Reply:
Tady mi to dává smysl - protože není jasné, proč by zrovna tady neměl být

Reply:
Přišlo by mi blbost tam ty message vyžadovat, ale pokud tam někde je a dává smysl, tak mi přijde, že tam může být :)

Reply:
No prave kdyz "nekde muze byt" tak se pak dostaneme do toho ze novy clovek nebude vedet jak ma vlastne ten test napsat. Jestli je to dulezity tak to dej jako komentar za to ale do asertu bych tu message nedaval

Reply:
OK